README.rst

```diff
@@ -42,6 +42,14 @@ The built documentation will be available at::
 
 Any good internet browser will be able to display the documentation (see also `Serving the documentation`_).
 
+To build the documentation into a single page HTML document execute the following command::
+
+  make singlehtml
```

Comment:
FWIW, I am still getting quite a few warnings when building the documentation:

```
$ make singlehtml
sphinx-build -b singlehtml -d build/doctrees   src build/singlehtml
Running Sphinx v2.4.4
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [singlehtml]: all documents
updating environment: [new config] 99 added, 0 changed, 0 removed
reading sources... [100%] reference/libcellml/libcellmlC03_interpretation_of_units
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB5.rst:12: WARNING: Unknown interpreted text role "libcellml_tutorials".
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB6.rst:12: WARNING: Unknown interpreted text role "libcellml_tutorials".
looking for now-outdated files... none found
pickling environment... done
checking consistency... /Users/Alan/Desktop/cellml-specification/src/reference/informative/informB9_reset1.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/informative/informC01_imports.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/informative/informC02_units_reference.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/informative/informC04_component_reference.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/informative/informC05_variable_reference.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/informative/informC07_effect_of_units_on_variables.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/informative/informC08_interpretation_of_mathematics.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/informative/informC09_interpretation_of_encapsulation.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/informative/informC10_interpretation_of_map_variables.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/informative/informC11_interpretation_of_variable_resets.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB1.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB12_math.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB15_connection.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB16_map_variables.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB2.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB3.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB4.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB5.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB6.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB7.rst: WARNING: document isn't included in any toctree
/Users/Alan/Desktop/cellml-specification/src/reference/libcellml/libcellmlB8.rst: WARNING: document isn't included in any toctree
done
preparing documents... done
assembling single document... reference/index_sectionA reference/formal_and_informative/specA1 reference/formal_and_informative/specA2 reference/formal_and_informative/specA3 reference/index_sectionB reference/formal_and_informative/specB01 reference/formal_and_informative/specB02 reference/formal_and_informative/specB03 reference/formal_and_informative/specB04 reference/formal_and_informative/specB05 reference/formal_and_informative/specB06 reference/formal_and_informative/specB07 reference/formal_and_informative/specB08 reference/formal_and_informative/specB09 reference/formal_and_informative/specB10 reference/formal_and_informative/specB11 reference/formal_and_informative/specB12 reference/formal_and_informative/specB13 reference/formal_and_informative/specB14 reference/formal_and_informative/specB15 reference/formal_and_informative/specB16 reference/index_sectionC reference/formal_and_informative/specC01_interpretation_of_imports reference/formal_and_informative/specC02_units_reference reference/formal_and_informative/specC03_interpretation_of_units reference/formal_and_informative/specC04_component_reference reference/formal_and_informative/specC05_variable_reference reference/formal_and_informative/specC06_interpretation_of_initial_values reference/formal_and_informative/specC07_effect_of_units_on_variables reference/formal_and_informative/specC08_interpretation_of_mathematics reference/formal_and_informative/specC09_interpretation_of_encapsulation reference/formal_and_informative/specC10_interpretation_of_map_variables reference/formal_and_informative/specC11_interpretation_of_variable_resets reference/index_sectionD done
writing... done
writing additional files... done
copying static files... ... done
copying extra files... done
dumping object inventory... done
build succeeded, 23 warnings.

The HTML page is in build/singlehtml.

Build finished. The HTML page is in build/singlehtml.
```

Reply:
All the "doc not in toctree" ones are known about - they're placeholders for my stuff.  I've turned off their inclusion so that we don't see "TODO" everywhere!

Reply:
hmm I'm not getting any.